styles.css

```diff
@@ -0,0 +1,250 @@
+.page-container {
+    height: 100vh;
+}
+
+/*Navbar*/
+.navbar {
+    background-color: var(--secondary-color);
+    box-shadow: var(--box-shadow);
+    position: fixed;
+    width: 100vw;
+    z-index: 1;
+}
+
+.title {
+    color: var(--primary-color);
+}
+
+.nav-links {
+    color: var(--text-color1);
+}
+
+.nav-cart {
+    color: var(--primary-color);
+}
+
+.searchbar-btn{
+    background-color: var(--white);
+    border: 0.5px solid black;
+    border-right: none;
+    padding: 0.625rem 1rem;
+}
+
+.searchbar-input {
+    border: 0.5px solid black;
+    border-left: none;
+    width: 35rem;
+    padding: 0.625rem 1.25rem;
+    outline: none;
+}
+
+.badge-container{
+    position: relative;
+    box-sizing: border-box;
+}
+
+.status-badge {
+    height: 1.25rem;
+    min-width: 1.25rem;
+    padding: 2px;
+    border-radius: 50%;
+    border: 2px solid var(--white);
+    display: flex;
+    color: var(--white);
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    font-size: .75em;
+    box-sizing: border-box;
+}
+
+.status-badge-number {
+    background-color: var(--primary-color);
+    border: none;
+    right: -1rem;
+    top: -0.7rem;
+}
+
+.searchbar-smallscreen {
+    display: none;
+}
+
+/*Main*/
+
+.main {
+    margin-top: calc(4.375rem + 0rem);
+}
+
+.home-image {
+    width: 100%;
+    height: 20rem;
+    object-fit: cover;
+    grid-column: 1/-1;
+    grid-row-start: 1;
+    border-radius: 1rem;
+}
+
+.home-text {
```

Comment:
Suggestion : Center Home Text 

![image](https://user-images.githubusercontent.com/90306959/152936287-db79b885-e4ed-4a8f-9b2a-c86b497bcd60.png)

Reply:
Sure will center the text. Thanks for reviewing.